week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, number) {
+  return "Hello, my name is" + name + "and I'm" + number + " years old";
+}
 const greeting = createLongGreeting("Daniel", 30);
 
 console.log(greeting);
```

Comment:
👍🏼
you could name the parameter **number** something that is more related to what it actually is i.e. **age**